po/af.po

```diff
@@ -8,11 +8,11 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: coding@endlessm.com\n"
-"POT-Creation-Date: 2017-09-07 13:39-0700\n"
+"POT-Creation-Date: 2017-09-07 22:49-0700\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Language-Team: Afrikaans (https://www.transifex.com/endless-mobile-inc/teams/9016/af/)\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Type: text/plain; charset=CHARSET\n"
```

Comment:
Hey Roddy, the change looks good, thanks for taking care of that. We slipped in the "CHARSET" again in the po files.

```
  MSGFMT de.gmo
de.po: warning: Charset "CHARSET" is not a portable encoding name.
                Message conversion to user's charset might not work.
```

Replacing that with "UTF-8" works then as expected.